src/main/java/com/rog/EShop/entity/Item.java

```diff
@@ -1,23 +1,97 @@
 package com.rog.EShop.entity;
 
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.Setter;
+import com.fasterxml.jackson.annotation.JsonBackReference;
 
 import javax.persistence.*;
-import java.util.List;
+import java.util.Objects;
+
 @Entity
-@Getter
-@Setter
-@EqualsAndHashCode
+@Table(name = "ITEMS")
 public class Item {
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "item_id_generator")
+    @SequenceGenerator(name = "item_id_generator", sequenceName = "item_id_seq")
```

Comment:
PostgreSQL automatically created sequence with other name.
But. as we discussed, let's manually create sequence for each table using Liquibase. https://docs.liquibase.com/change-types/create-sequence.html